### DISCUSSION TITLE
Unable to access in the discord and question about dogecoind

### DISCUSSION THREAD

**Discussion by GaloisField2718** [dogecoin/dogecoin#3281] [General]
Hello everyone,

I would post my question on the discord but after have been verified I only see the text chat dogecoin without be able to access.

I would like to know how can I dogecoind on MacOS. I installed the GUI (Qt) but instead I want to have dogecoind.
I didn't find anything on the git about `dogecoind` on MacOS. Do you have any information about this ?

Thanks in advance :)

**Comment by patricklodder** [comment]
Hi,

We currently do not package dogecoind for macOS in the release binaries. We can look into providing an additional archive file that includes dogecoind / dogecoin-cli in the release process; this has been requested once before.

In the interim you can try to compile it yourself, but do note that I am not aware of anyone being able to compile a working, backward-compatible wallet in at the moment, so you will run into issues if you would ever want to restore your wallet.dat on another platform, and you'd be at the mercy of both brew maintainers and Oracle for providing a package that can read the database with your keys.

If you're just looking for the RPC, note that if you add `server=1` to `~/Library/Application\ Support/Dogecoin/dogecoin.conf`, you can serve the RPC using Qt too (this is by far the safest method, because you will not have an incompatible wallet file)
